### ISSUE TITLE
Any plan for a Java Backend ?

### ISSUE THREAD

**Issue by zzj1213** [bokeh/bokeh-scala#29]
Java is a lot more popular than Scala even though need a compile.  Java has much better tooling like Eclipse。

**Comment by bryevdv** [comment]
There are no immediate plans. I would love to see a Java binding for Bokeh. But the reality is that the small core team is already over-committed and is struggling to keep up. Any work on a Java binding would have to either led by and performed by motivated new contributors, or someone or some organization would have to decide it worth to fund the work so that developers could be hired to perform the work. If you are in a position to make either of those things happen, I'm definitely happy to discuss more.

**Comment by zzj1213** [comment]
Thanks for the reply.  Yes, we are interested to add Java supports.  We are currently using Eclipse RAP (www.eclipse.org/rap) as the UI Framework.   Eclipse RAP is all Java for Web App, thus seems fitting Bokeh's idea very well.

**Comment by bryevdv** [comment]
@zzj1213 I agree `bokeh.java` could be very useful. If you are planning to develop it yourself, please let me know if you'd like into be available under a repo in the "Bokeh" GitHub organization. Assuming you are planning to release it Open Source, I think that would be the best place for it. Alternatively, if you are interested in a commercial consulting engagement to fund Continuum Analytics to develop it, please contact me at the email address in my GH profile, and I will put you in touch with the right people for that discussion.
